src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11918,6 +11893,9 @@ void Player_Action_8084B9E4(Player* this, PlayState* play) {
             if (LinkAnimation_OnFrame(&this->skelAnime, 11.0f)) {
                 func_80832698(this, NA_SE_VO_LI_PUSH);
             }
+#if !OOT_DEBUG
+            if (1) {}
+#endif
```

Comment:
should this be `if (!OOT_DEBUG){}` ?

Reply:
It seems like `#if` is slightly more common these days at least (I found 3 instances of `#if` around `if (1) {}`, and only 1 instance of `if (OOT_DEBUG) {}` or `if (!OOT_DEBUG) {}` in z_play.c)

Reply:
Does is still match both versions? I like having it just as 1 line instead of 3 because it's a fake match anyway, so it takes up less space. But sometimes `if (0) {}` isn't enough to fix regalloc instead of having `if (1) {}` removed

Reply:
it does match in this case

Reply:
i pushed it since I also like the fake match being compact